src/pls.py

```diff
@@ -95,7 +95,7 @@ def saveModel(self,filename):
 
         np.save(f,self.autoscale)
         
-        for a in range(self.Am):
+        for a in xrange(self.Am):
```

Comment:
`xrange` is a generator while `range` makes a lists and allocates it in memory. `xrange` is always faster ( more noticeable with big numers)
![screenshot from 2017-09-08 14-35-01](https://user-images.githubusercontent.com/9404338/30211893-fc206864-94a2-11e7-8df4-d8f36e229f7e.png)